Gemfile

```diff
@@ -5,6 +5,8 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
+# Auth devise
+gem 'devise'
```

Comment:
when adding gems we usually specify the version to avoid problems when updating.
You could copy paste the `gemfile` version from here https://rubygems.org/gems/devise

Reply:
and what happens when it's updated? Do you change all the version on projects gemfiles?

Reply:
No, we just update it when we think it's necessary, rather than when there is an available update